tests/common/fixtures/elasticsearch.py

```diff
@@ -17,10 +18,19 @@ def es_client(delete_all_elasticsearch_documents):
     return _es_client()
 
 
+@pytest.fixture
+def es_connect():
+    # TODO handle deleting things out of this connection's index as
+    # the `es_client` fixture does
+    search.connect(hosts=[ELASTICSEARCH_URL])
```

Comment:
Does this need to be a fixture? If it's just a helper function for the `init_elasticsearch` fixture to call then it could just be a private helper function (or the `search.connect()` line could just go in `init_elasticsearch()`).

Reply:
Perhaps premature optimization on my part: I anticipate that we'll need a wrapper once things start getting more "complex" in the lifecycle. Plus there could be some case in which a test might want access to just the "connect" part of the lifecycle. I dunno. Should I kill it for now?